contributor/es/01-introduction-article-es.asciidoc

```diff
@@ -0,0 +1,63 @@
+
+=== Relación con otros roles
+
+Como Colaborador en una comunidad de InnerSource, interactuará con otras personas que desempeñan otros roles de InnerSource, como https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] o https://innersourcecommons.org/learn/learning-path/product-owner[_Product Owner_] y posiblemente con otros colaboradores.
+A veces, estos roles pueden ser desempeñados por la misma persona, como Trusted Committer y Product Owner en pequeños proyectos.
+
+Esta sección le brinda una muy breve descripción general de los otros dos roles, pero nos gustaría animarle a que lea el https://innersourcecommons.org/learn/learning-path/trusted-committer[introductory article of the Trusted Committer role], y le recomendamos que lea también el artículo https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[Lowering the barriers to entry] antes de profundizar en los detalles del rol de Colaborador en esta sección.
+También puede ver los videos (https://innersourcecommons.org/learn/learning-path/trusted-committer[introduction], https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[lowering the barriers to entry]) en lugar de leer los artículos.
+
+==== Trusted Committer
+
+Un Trusted Committer será su anfitrión durante su estancia en la comunidad anfitrión.
+Son los responsables del repositorio de código del proyecto y acercarán su contribución a la producción una vez sea aceptada.
+Su función es guiarle para contribuir a su comunidad. Ellos pueden ayudarlo directamente o proporcionarle información que le permita continuar por sí mismo. Esta información podría ser, reglas internas establecidas para revisiones, plantillas de propuestas para cambios más importantes, referencias a la documentación o secciones de código relevantes para su contribución.
+
+También deben preocuparse por la calidad del producto, la sostenibilidad y la evolución del proyecto desde una perspectiva técnica y general, por reducir la barrera para hacer contribuciones para todos, así como por el cuidado de su comunidad en general.
+Cuidar de la comunidad implica mantenerla saludable, elevarla a ella y a sus participantes, y defender sus necesidades en su organización.
+
+==== Product Owner
+
+El rol de https://innersourcecommons.org/learn/learning-path/product-owner[_Product Owner_] tiene cierta similitud con el rol de product owner de su proyecto.
+Sin embargo, existen diferencias: dependiendo del tamaño del proyecto, este rol a menudo lo desempeña la misma persona que actúa como responsable de confianza.
+En proyectos más grandes, o en equipos que solo usan parcialmente InnerSource para satisfacer sus necesidades al aceptar contribuciones, es probable que este rol lo desempeñe una persona separada de un trusted committer.
+
+Su interacción con el rol de product owner probablemente se centrará en determinar el ajuste de su contribución al producto general y su hoja de ruta.
```

Comment:
"ajuste al" feels more technical (polishing rough edges) while the interaction here (with the PO) is about strategic alignment. "alineamiento con" is less misleading.

Reply:
👍